webpack/src/Components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, EmptyState as PfEmptyState } from 'patternfly-react';
+import { useSelector, useDispatch } from 'react-redux';
+import { selectEmptyStateHeader } from './EmptyPageSelectors';
+import { AddEmptyStateContent } from './EmptyStateActions';
+
+const EmptyState = ({ description }) => {
+  const dispatch = useDispatch();
+  const header = useSelector(selectEmptyStateHeader);
+
+  useEffect(() => {
+    dispatch(AddEmptyStateContent('Foreman Plugin Template - React Page'));
+  });
+
```

Comment:
What did we decide about keeping the `index.js` convention of core,
should we move the `redux` login into the `index.js` file so the `EmptyState.js` file will contain only the React view part?